generator/transit_generator.cpp

```diff
@@ -230,14 +213,21 @@ void DeserializerFromJson::operator()(FeatureIdentifiers & id, char const * name
   id = FeatureIdentifiers(osmId.EncodedId() /* osm id */, it->second[0] /* feature id */);
 }
 
-void BuildTransit(string const & mwmPath, string const & osmIdsToFeatureIdPath,
-                  string const & transitDir)
+void DeserializerFromJson::operator()(StopIdRanges & rs, char const * name)
```

Comment:
Do we need this?

Reply:
Yes. We do. We need to read vector of stopIds from json. And to set to Gate vector of vector of stopIds. For detail explanation please have a look at the descr of the PR and at the ticket: https://jira.mail.ru/browse/MAPSME-5858

Reply:
But current version of DeserializerFromJSON() should call operator() for vector, what's wrong?

Reply:
Yes. The current version of DeserializerFromJSON() calls operator() for vector. And then StopIdRanges is initialized with the vector. So, after that there's vector of vectors with one item in StopIdRanges (in Line). 

Then vector of vectors with one items is serialize to mwm.

But why not to save just a vector instead? Because it's a preparation for cutting (clipping) transit graph by mwm borders. In next PRs line may be cut by mwm border into several parts (into several stop id ranges). So that's why we need to have vector of vectors in Line.

Did I answer your question?

Reply:
Okay, see now.